pages/common/curl.md

```diff
@@ -16,7 +16,7 @@
 
 `curl -O -L -C - {{http://example.com/filename}}`
 
-- Send form-encoded data (POST request of type `application/x-www-form-urlencoded`):
+- Send form-encoded data (POST request of type `application/x-www-form-urlencoded`), try -d {{'@filename}} or -d {{'@-'}} to read from STDIN:
```

Comment:
We should probably use a period, and then start a new sentence. Perhaps something like
"Use `-d @file_name` or `-d @'-'` to read from STDIN"

Reply:
Updated comment to use a code fence to aid clarity :-)